src/models/usersDao.js

```diff
@@ -19,6 +29,30 @@ const signUp = async (email, password, points) => {
   }
 };
 
+const logIn = async (email) => {
```

Comment:
dao의 함수명은 어떤 행위를 하는지를 나타내 주세요.
```suggestion
const getUserByEmail = async (email) => {
```